src/components/VideoCardHorizontal/VideoCardHorizontal.jsx

```diff
@@ -28,6 +37,10 @@ const VideoCardHorizontal = ({
     }
     return navigate(`/video/${_id}`);
   };
+
+  let isVideoInWatchLater =
+    watchLaterVideos.findIndex((item) => item._id == _id) == -1 ? false : true;
```

Comment:
why using find index and you can do this with some right?
as it will directly give you the boolean value